build-configs/themes/integreat/index.js

```diff
@@ -14,8 +14,8 @@ const theme: ThemeType = {
 }
 
 export const darkTheme: ThemeType = {
-  colors: integreatDarkColors,
-  ...theme
+  ...theme,
+  colors: integreatDarkColors
 }
```

Comment:
wat? :laughing: Didn't this already work? :thinking:

Reply:
I think it did work but my changes and moving this to separate build configs broke it. Working now again though.